### ISSUE TITLE
BERT-L is not using cached model

### ISSUE THREAD

**Issue by nmb-paperspace** [gradient-ai/Graphcore-HuggingFace#2]
The `introduction_to_optimum_graphcore.ipynb` notebook is not using the cached model like the one on the PyTorch runtime (https://github.com/zzweig/pytorch-graphcore/blob/main/get-started/Fine-tuning-BERT.ipynb), and so takes ~ 10 minutes to build the graph.

**Comment by nmb-paperspace** [comment]
Deniz commented:

> Same with https://github.com/zzweig/huggingface-graphcore/issues/15 , @internetoftim will create a diff.
